techniques/failures/F38.html

```diff
@@ -5,7 +5,7 @@
                         should be ignored by AT. If there is no alt attribute at all assistive
                         technologies are not able to ignore the non-text content. The alt attribute
                         must be provided and have a null value (i.e., alt="" ) to avoid a failure of this Success Criterion. </p>
```

Comment:
```suggestion
                        must be provided and have a null value (i.e., <code>alt=""</code> ) to avoid a failure of this Success Criterion. </p>
```

Found this cheeky little guy trying to escape.